### ISSUE TITLE
BETA pill on search modal is not accessible by keyboard

### ISSUE THREAD

**Issue by frakic** [vector-im/element-web#21255]
### Steps to reproduce

1. Where are you starting? What can you see?
 -I've opened the Search modal (Ctrl+K) and hit TAB in order to get to the BETA pill in the bottom left corner
2. What do you click?
 -I'm hitting the return key in order to open Labs but nothing is happening
3. More steps…

#### What did you expect?
Everything should be accessible by keyboard.
#### What happened instead?
Hitting return/space keys while focused on the BETA pill does nothing.

### Operating system

Windows 10 Home 21H1

### Browser information

Firefox 97.0.1 (64-bit)

### URL for webapp

https://develop.element.io

### Application version

Element version: ce7094182d2b-react-ac362340688b-js-b782dee2ef9f Olm version: 3.2.8

### Homeserver

matrix.org

### Will you send logs?

No
